### DISCUSSION TITLE
[Kavita] Idea / Feature Submission Option to ignore comicinfo.xml

### DISCUSSION THREAD

**Discussion by fonephreak** [Kareadita/Kavita#2854] [Ideas]
### Idea Description

An option in libraries to not use comicinfo.xml. So if that library is scanned the comicinfo can be used or ignored. Komga has a similar option when adding libraries and it helps a lot with adding mangas but I rather use Kavita since it handles ebooks better.

This has only mattered to me because of recent updates to Kavita. Some of the mangas in my manga library have comicinfo.xml files in the cbz but not all which didn’t matter before. However now because of reading the file during scans it spilts 1 manga into 2 mangas which almost doubled my library count with duplicates with chapters split between. Which messes up read count and metadata I entered into Kavita by hand.

### Idea Category

User Experience

### Duration of Using Kavita

Since the beginning

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I think this indicates a bug. Please create an issue with more details so I can fix it.

**Reply by fonephreak** [reply]
Ok I’ll do that when add new chapters and the issue happens again. I have been decompressing each affected cbz file then deleting the comicinfo.xml then compressing it back to a cbz file then rescanning the library . Which was a pain for 200+ mangas with more than 20 chapters affected each. I turned off auto update and import once a week currently so I’m not editing files everyday. That is why I asked it there is a way to tell Kavita to ignore the file for certain libraries.

**Reply by Cducharme84** [reply]
> Ok I’ll do that when add new chapters and the issue happens again. I have been decompressing each affected cbz file then deleting the comicinfo.xml then compressing it back to a cbz file then rescanning the library . Which was a pain for 200+ mangas with more than 20 chapters affected each. I turned off auto update and import once a week currently so I’m not editing files everyday. That is why I asked it there is a way to tell Kavita to ignore the file for certain libraries.

[Comicbox](https://github.com/ajslater/comicbox) has a command line delete option that sounds like it’d do the trick for you. It’s fast since it just deletes the xml no need for full extraction and repack.

**Reply by fonephreak** [reply]
I organize my manga library based off of file structure and not data from the comicinfo.xml file. When a new chapter is added that has a comicinfo.xml file Kavita prioritizes that over my existing file structure I’m using thus creating a new manga. That’s why I have requested an option to toggle the checking of the comicinfo.xml file off for libraries for people who prefer not to use it and rather use file structure. 

In the end I gave up because my library kept messing up, I just created a powershell script that scans for and removes the comicinfo.xml file from all the CBZ files in my manga directory

**Comment by majora2007** [comment]
Similar idea, which indicates providing it as a library option would be the best approach:
https://github.com/Kareadita/Kavita/discussions/3695

**Comment by majora2007** [comment]
I pushed out the ability to turn off metadata (for any library). Would love some feedback, it's on the nightly branch. Otherwise it will go out (without feedback) in v0.8.7 and any feedback will have to be addressed in the next release cycle. 

https://github.com/Kareadita/Kavita/pull/3872
